src/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PATH2S</title>
+    <script src="https://kit.fontawesome.com/35dafefa29.js"
+      crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header>
+      <div class="logo">
+        <p>PATH2S</p>
+      </div>
+
+      <div class="header-links">
+        <a href="">Home</a>
+        <a href="">Rockets(<span class="launch-count"></span>)</a>
+        <a href="">Programs</a>
+        <a href="">About</a>
+        <div class="media-links">
+          <a href="https://www.instagram.com/jay_iz_just_awesome/"><i class="fab fa-instagram"></i></a>
+          <a href="https://web.facebook.com/adesanya.juwon.5"><i class="fab fa-facebook"></i></a>
+          <a href="https://twitter.com/BB_Ludidi"><i class="fab fa-twitter"></i></a>
+          <a href="https://github.com/Becky449"><i class="fa-brands fa-github"></i></a>
+        </div>
+      </div>
+    </header>
+    <div id="launch-modal">
+      <div class="launch-card">
+        <button class="close">x</button>
+        <div class="launch-img">
+          <div class="content">
+            <h1></h1>
+            <div class="details">
+              <ul>
+                <li>
+                  <span class="bold">Launch Provider:</span>
+                  <span class="lsp"></span>
+                </li>
+                <li>
+                  <span class="bold">Mission Type:</span>
+                  <span class="mission"></span>
+                </li>
+                <li>
+                  <span class="bold">Pad:</span>
+                  <span class="pad"></span>
+                </li>
+                <li>
+                  <span>Location:</span>
+                  <span class="location"></span>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div id="comment">
+          <h2>Comments (<span class="comments-count"></span>)</h2>
+          <ul class="commentList">
+          </ul>
+          <div class="entry">
+            <h2>Add a Comment</h2>
+            <form>
+              <input id="txt-visitor" type="text" placeholder="Your name" />
+              <input id="txt-comment" type="text" placeholder="Your comments" />
+              <input id="launch-id" type="hidden" />
+              <input id="btn-save-comment" type="button" value="Comment" />
+            </form>
+          </div>
+        </div>
+        <div id="reserve">
+          <h2>Reservation (<span class="reserve-count"></span>)</h2>
```

Comment:
- [ ] I think it's better to create a specific function to calculate the reserve counter. A counter function should look for specific DOM elements and make the counting based on what is actually displayed on the page. Don't forget these count functions need to be covered with unit tests using Jest. Don't forget the case when the count is zero.